frontend/pages/hosts/ManageHostsPage/_styles.scss

```diff
@@ -246,7 +233,6 @@
 
     .Select-menu-outer {
       width: 364px;
-      max-height: 325px;
```

Comment:
Seems like you nudged this upward previously, but weren't the original one to add this, and prior to your most recent fix the code had sat there for 3 years, so not surprised the styling is out of date. Fair enough to clean this up :)

Reply:
Yeah, the max height is being used and set in a lot of places but here it just cuts off the options >.<